$NetBSD: patch-src_meson.build,v 1.1 2023/06/27 10:42:07 riastradh Exp $

When cross-compiling, let the caller (the pkgsrc Makefile) specify the
exact path to wayland-scanner rather than flailing around with
pkgconfig or cmake to (not) find it.

--- src/meson.build.orig	2022-06-30 21:59:11.000000000 +0000
+++ src/meson.build
@@ -78,8 +78,7 @@ if get_option('scanner')
 endif
 
 if meson.is_cross_build() or not get_option('scanner')
-	scanner_dep = dependency('wayland-scanner', native: true, version: meson.project_version())
-	wayland_scanner_for_build = find_program(scanner_dep.get_variable(pkgconfig: 'wayland_scanner'))
+	wayland_scanner_for_build = find_program(get_option('cross_scanner'))
 else
 	wayland_scanner_for_build = wayland_scanner
 endif
